notesystem/modes/convert_mode.py

```diff
@@ -275,15 +286,20 @@ def _convert_file(self, in_file: str, out_file) -> None:
     def _convert_dir(self, in_dir_path: str, out_dir_path: str) -> None:
         """Converts all the markdown files in a directory (and subdirectory) to html
 
-        Using pandoc all the files in the `in_dir_path` directory (and it's subdirectories)
-        are converted to html and saved in the out_dir_path and if nessesary in their correct subfolder.
+        Using pandoc all the files in the `in_dir_path` directory
+        (and it's subdirectories) are converted to html and saved
+        in the out_dir_path and if nessesary in their correct subfolder.
+
         So the folder structure and filenames stay the same.
 
-        If the out_dir_path does not exits, it gets created and so do the subdirectories.
+        If the out_dir_path does not exits, it gets created
+        and so do the subdirectories.
 
         Arguments:
-            in_dir_path {str} -- The directory where all the files that need to be converted are located
-            out_dir_path {str} -- The directory where all the converted files will be saved.
+            in_dir_path {str}  -- The directory where all the files that need
+                                    to be converted are located
+            out_dir_path {str} -- The directory where all the converted files
+                                    will be saved.
```

Comment:
Use indent to the exact same, or one more but not both!

```suggestion
            in_dir_path {str}  -- The directory where all the files that need
                                  to be converted are located
            out_dir_path {str} -- The directory where all the converted files
                                  will be saved.
```